spec/features/memberships/invite_user_to_their_first_organisation_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'support/invite_use_case'
```

Comment:
The spec I was just working on:
spec/features/team/accept_an_invitation_and_sign_up_spec.rb

Seems to assert on this too.  Wonder if we can combine these specs?
I know there is a large name refactor pending.

Reply:
Theres loads we can do here. Lets leave it for now, fix the edge cases, then improve the specs.